src/fast_array_utils/conv/__init__.py

```diff
@@ -52,6 +55,8 @@ def to_dense(
     ----------
     x
         Input object to be converted.
+    order
+        The order of the output array: ``C`` (row-major) or ``F`` (column-major). ``K`` and ``A`` derive the order from ``x``.
```

Comment:
I would add more info here i.e., what you say in the PR comment

Reply:
done!